app/posts/manage-teacher-training-applications/2021-03-18-self-service-registration.md

```diff
@@ -0,0 +1,436 @@
+
+We removed the ‘Do you want to open courses at the training provider?’ question and went straight to asking for contact details.
+
+We did not use the task list pattern as we decided it added unnecessary complications.
+
+### What we did and why
+
+{{ appFigure({
+  image: {
+    file: "self-service-registration-v3-flow.png",
+    alt: "Iteration 3 user flow"
+  },
+  caption: "Figure 9: Iteration 3 user flow"
+}) }}
+
+The start page listed the SDs we needed contact details for. We only asked about SDs which are not already using Manage, reducing the effort required by the HEIs to complete the process.
+
+We also listed the SDs which are already using Manage, so that the user would not think we’d missed them out.
+
+{{ appFigure({
+  image: {
+    file: "self-service-registration-v3-01.png",
+    alt: "Start page inviting providers to manage teacher training applications"
+  },
+  caption: "Figure 10: Start page inviting providers to manage teacher training applications"
+}) }}
+
+For each of the SDs which were not on Manage, we included a list of people who already had accounts in the Publish service. This was intended to further reduce the effort needed to complete the form.
+
+Users could either choose one of these people or add new contact details.
+
+{{ appFigure({
+  image: {
+    file: "self-service-registration-v3-02a.png",
+    alt: "Question page asking ‘Who do you want to invite to manage applications at the training provider?’ and showing pre-defined list of people"
+  },
+  caption: "Figure 11: Question page asking ‘Who do you want to invite to manage applications at the training provider?’ and showing pre-defined list of people"
+}) }}
+
+{{ appFigure({
+  image: {
+    file: "self-service-registration-v3-02b.png",
+    alt: "Question page asking ‘Who do you want to invite to manage applications at the training provider?’ and showing the ‘someone else’ option fields"
+  },
+  caption: "Figure 12: Question page asking ‘Who do you want to invite to manage applications at the training provider?’ and showing the ‘someone else’ option fields"
+}) }}
+
+{{ appFigure({
+  image: {
+    file: "self-service-registration-v3-02c.png",
+    alt: "Question page asking ‘Who do you want to invite to manage applications at the training provider?’ only showing name and email fields as there are no known people in the organisation"
+  },
+  caption: "Figure 13: Question page asking ‘Who do you want to invite to manage applications at the training provider?’ only showing name and email fields as there are no known people in the organisation"
+}) }}
+
+The data sharing agreement remained the same as in Manage.
+
+{{ appFigure({
+  image: {
+    file: "self-service-registration-v3-03.png",
+    alt: "Data sharing agreement"
+  },
+  caption: "Figure 14: Data sharing agreement"
+}) }}
+
+We removed mention of the DSA from the ‘check your answers’ page since there was already a confirmation check box at the end of the DSA itself.
```

Comment:
sometimes the post says ‘check your answers’ and sometimes check answers without quotes.

I would suggest: `‘check answers’ page` throughout.

Reply:
I have updated as suggested.